webpack/app/js/datepicker.js

```diff
@@ -129,17 +90,19 @@ export default class DatePicker {
     // this event is needed to get right translation
     $(window).on('load', function() {
       let onSelect = function(dateText, inst) { 
-        this.year = inst.selectedYear;
-        this.month = inst.selectedMonth;
-        this.day = parseInt(inst.selectedDay);
-        this.update();
+        let year = inst.selectedYear;
+        let month = inst.selectedMonth + 1;
+        let day = parseInt(inst.selectedDay);
+        // https://stackoverflow.com/a/19374679
+        window.location = `${this.agendaURL}/${year}-${zfill(month)}-${zfill(day)}`;
```

Comment:
the answer you're referring uses a cleaner approach using  a date format.

Reply:
I can't use the same strategy because the ajax is waiting a different date format